### ISSUE TITLE
Disable legend toggle zoom behavior

### ISSUE THREAD

**Issue by Eldair** [amcharts/amcharts4#678]
Hi,

Is it possible to have legend toggle behavior but without zoom side-effect?

**Comment by martynasma** [comment]
Zoom side effect?

Do you mean that if toggled off series gets animated to zero line and if that affects axis scale, it also gets a kind of animation to new scale?

**Comment by martynasma** [comment]
Gotcha.

I'm afraid there's currently no such setting. A `ValueAxis` will try to adjust its scale to the actual visible range of values, unless you have `min`/`max` set.

So, the workaround would be to fix those when the chart loads initially, so they do not change during subsequent interactions with the chart, such as hiding series.

E.g.:

```
chart.events.on("ready", function(ev) {
  valueAxis.min = valueAxis.minZoomed;
  valueAxis.max = valueAxis.maxZoomed;
});
```

Here's an example:
https://codepen.io/team/amcharts/pen/LMGXpJ?editors=0010

Hope you find this useful.

**Comment by Eldair** [comment]
Thanks for the answer, I think I'll use the suggested code until that maxZoomFactor is fixed.
Also thanks for that event :D I was trying to use it but wasn't sure on which object to set it up.

**Comment by zeroin** [comment]
The maxZoomFactor problem was fixed in 4.0.11
